### DISCUSSION TITLE
[puthon] I would like to ask what requirements the package that can be installed needs to satisfied.

### DISCUSSION THREAD

**Discussion by cat-my-earn** [thebaselab/codeapp#458] [Q&A]
I can’t install reportlab by pip. I would like to ask what requirements the package that can be installed needs to meet. And I would like to ask if it is possible to support the package which can edit PDF in the future.

**Comment by bummoblizard** [comment]
Hi. There are few conditions:
- Pure Python code (does not call native code)
- Do not spawn subprocess
- Or ported and bundled in the app by the developer. 

I would love to work on this if there's a popular demand for it.
